keanu-python/keanu/algorithm/sampling.py

```diff
@@ -97,6 +114,28 @@ def sample(net: BayesNet,
            down_sample_interval: int = 1,
            plot: bool = False,
            ax: Any = None) -> sample_types:
+    """
+    :param net: Bayesian Network containing latent variables.
+    :param sample_from: Vertices to include in the returned samples.
+    :param sampling_algorithm: The posterior sampling algorithm to use.
+        Options are :class:`keanu.algorithm.MetropolisHastingsSampler` and :class:`keanu.algorithm.NUTSSampler`.
+        If not set, :class:`keanu.algorithm.MetropolisHastingsSampler` is chosen with 'prior' as its proposal distribution.
+    :param draws: The number of samples to take.
+    :param drop: The number of samples to drop before collecting anything.
+        If this is zero then no samples will be dropped before collecting.
+    :param down_sample_interval: Collect 1 sample for every `down_sample_interval`.
+        If this is 1 then there will be no down-sampling.
+        If this is 2 then every other sample will be taken.
+        If this is 3 then 2 samples will be dropped before one is taken.
```

Comment:
I'd suggest adding "and so on" on a new line, just to make it clear that 1,2,3 are not the only options!